### DISCUSSION TITLE
bigfix/11.0/inventory/Inventory/planinconf/c_supp_dev

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#84] [Q&A]
# bigfix/11.0/inventory/Inventory/planinconf/c_supp_dev

The tables in this section provide detailed information about supported managed devices for BigFix Inventory version 10.

https://help.hcl-software.com/bigfix/11.0/inventory/Inventory/planinconf/c_supp_dev.html

**Comment by JoeBrunsTR** [comment]
Would be prudent to list someplace, top maybe of this page, the last time and date this page was published.

**Comment by syeda-git** [comment]
Thank you for the feedback. We will plan to incorporate it in the doc pages.

**Comment by JoeBrunsTR** [comment]
Would be ideal to allow a customer to download all of the OS data into a CSV file.
